lib/hammer_cli_foreman/ssh.rb

```diff
@@ -39,7 +47,8 @@ def execute
       ssh_options = { :user => user, :auth_methods => ['publickey'] }
       ssh_options[:keys] = [identity_file] unless identity_file.to_s.empty?
 
-      Net::SSH::Multi.start(:on_error => :warn) do |session|
+      Net::SSH::Multi.start(:concurrent_connections => limit, :on_error => :warn) do |session|
+        puts _("executing on #{limit} concurrent host(s)") if limit.to_i > 0
```

Comment:
I'm not aware about how important this information is for the user, but FYI there is also option to just log it with `logger.info(_(.....))`